src/test/java/spoon/test/template/testclasses/EnumAccessTemplate.java

```diff
@@ -13,15 +12,15 @@ public void method() throws Throwable {
 	}
 	
 	@Parameter("_AnEnum_")
-	CtTypeReference<?> anEnum;
+	String __AnEnum_;

```

Comment:
Try to compile it. If it pass then relax contract. If it is not compilable then keep contract as it is. My usage, which I show you soon, is not with conflict with this PR so it is ok.